style.css

```diff
@@ -11,15 +11,26 @@ p {
 
 body {
   background-color: white;
-  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    "Open Sans",
+    "Helvetica Neue",
+    sans-serif;
   color: black;
   font-size: 17px;
   padding: 15px 10px;
   text-align: center;
 }
```

Comment:
Please review your fonts (family, size, and weight), try to adjust your margins (top, right, bottom, and left), and reduce the line height in between your text.